homeassistant/components/rfxtrx/config_flow.py

```diff
@@ -145,6 +151,10 @@ async def async_step_prompt_options(self, user_input=None):
                 CONF_AUTOMATIC_ADD,
                 default=self._config_entry.data[CONF_AUTOMATIC_ADD],
             ): bool,
+            vol.Optional(
+                CONF_PROTOCOLS,
+                default=self._config_entry.data.get(CONF_PROTOCOLS, []),
+            ): cv.multi_select(RECV_MODES),
             vol.Optional(CONF_EVENT_CODE): str,
```

Comment:
I do t fully remember how an optional multiselect works. Can we get back to the non set device default mode in any way now? If you deselect all protocols i suspect it will be an empty list rather that none?

Reply:
Yes, it will be an empty list rather than none, but that is handled when passing the modes to the library. An empty list will be swapped out for None so that the library handles it correctly.

Reply:
I actually swapped out the impl now, so it will change [] -> None if it comes back from the form, but you can still set an empty array in the yaml config if you really want to use no protocols, just to close down the edge case.